specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/sqlPool.json

```diff
@@ -7623,6 +7688,86 @@
           }
         }
       }
+    },
+    "DataWarehouseQueriesSteps": {
+      "description": "The response to a list datawarehouse queries.steps",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/DataWarehouseQueriesStepsProperties",
+          "description": "Resource properties."
+        }
+      }
+    },
+    "DataWarehouseQueriesStepsProperties": {
+      "description": "Properties of a datawarehousequeries.",
+      "type": "object",
+      "properties": {
+        "queryId": {
+          "description": "Requested QueryId. of datawarehousequeries",
+          "type": "string",
+          "readOnly": true
+        },
+        "startTime": {
+          "description": "The start time for queries datawarehousequeries",
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true
+        },
+        "endTime": {
+          "description": "The end time for queries",
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true
+        },
+        "duration": {
+          "description": "The duration of requested queries.",
+          "type": "number",
+          "format": "int32",
+          "readOnly": true
+        },
+        "status": {
+          "description": " Status of dataWarehouseQueries.",
+          "type": "string",
+          "readOnly": true
+        },
+        "queryText": {
+          "description": " QueryText Message",
+          "type": "string",
+          "readOnly": true
+        },
+        "stepIndex": {
+          "description": "dataWarehouseQueries step index",
+          "type": "string",
+          "readOnly": true
+        },
+        "operationType": {
+          "description": "Type of operation",
+          "type": "string",
+          "readOnly": true
+        },
+        "distributionType": {
```

Comment:
What are the possible values of distributionType?

Reply:
As per the response from Sql Team, Included properties
![image](https://user-images.githubusercontent.com/109329038/185033359-8b0f5c86-77d0-40d5-a599-f2c2ef736169.png)

Reply:
@bansalaseem added Values for distributionType